GUIDE.md

```diff
@@ -0,0 +1,105 @@
+# Editing mappings
+
+## Starting up
+
+After running a mappings task (either `mappings` or `mappingsUnpicked` via Gradle), you'll be greeted with this imposing Swing GUI:
+![an image of enigma's main window, devoid of life. the "deobfuscated classes", "obfuscated classes", and "structure" dockers are open on the sides, and nothing is displayed in the centre of the screen.](https://github.com/ix0rai/quilt-mappings/assets/66223394/fc569c2c-7b6e-4d22-9199-dd9b52afd8ca)
+
+The first thing you want to draw your attention to is the left of the screen, where you'll find the "Obfuscated Classes" and "Deobfuscated Classes" dockers.
+Obfuscated Classes will contain classes where the top-level name remains unmapped, and "Deobfuscated Classes" the mapped ones.
+Alternatively, you can use the merged "Classes" docker to view all classes at once, by pressing the button on the far left.
+Dockers can be resized and their titles dragged around so you can find the best setup for you!
+
+All three dockers on the left serve the same purpose: showing you the classes in the project, just like you'd see in your IDE.
+Each class has a little icon to show how well-mapped it is: a green check for done, a yellow dot for in progress, and a red x for untouched.
+Double-click on packages and classes to open them. You can also right-click to see a context menu with some shortcuts to help you map faster!
+When you open a class, it will be decompiled and the source code shown in the main view in the centre.
```

Comment:
```suggestion
When you open a class, it will be decompiled and the source code shown in the main view in the center.
```
"center" is the more conventional American English spelling.